adam/curriculum/verbs_with_dynamic_prepositions_curriculum.py

```diff
@@ -764,6 +765,95 @@ def _x_rolls_y_over_under_z_template(
     )
 
 
+# FALL templates
+
+
+def _fall_on_template(
+    theme: TemplateObjectVariable,
+    goal_reference: TemplateObjectVariable,
+    background: ImmutableSet[TemplateObjectVariable],
+    *,
+    syntax_hints: ImmutableSet[str],
+) -> Phase1SituationTemplate:
+    return Phase1SituationTemplate(
+        f"{theme.handle}-falls-(down)-on-{goal_reference.handle}",
+        salient_object_variables=[theme, goal_reference],
+        background_object_variables=background,
+        actions=[Action(FALL, argument_roles_to_fillers=[(THEME, theme)])],
+        after_action_relations=flatten_relations(on(theme, goal_reference)),
+        constraining_relations=[bigger_than(goal_reference, theme)],
```

Comment:
I know we've been inconsistent with this in the past but we should probably try to always use `flatten_relations` in a field which expects an `Iterable[Relation]`